provisioner/influx-series.js

```diff
@@ -39,6 +39,78 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
   },
 });
 
+// Store the times when a spot request is fulfilled
+module.exports.spotRequestsFulfilled = new base.stats.Series({
+  name: 'AwsProvisioner.SpotRequestsFulfilled',
+  columns: {
+    provisionerId: base.stats.types.String,
+    region: base.stats.types.String,
+    az: base.stats.types.String,
+    instanceType: base.stats.types.String,
+    workerType: base.stats.types.String,
+    id: base.stats.types.String,
+    instanceId: base.stats.types.String,
```

Comment:
Do we really want `id` and `instanceId` ?
These won't translate to tags in 0.9 because they have a much higher cardinality.

That said I still think we can have them... They are just slow to query by, takes up a lot of space and is hard to use. But if we have use-cases it probably not a problem.